onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -139,7 +142,7 @@ TEST_P(ModelTest, Run) {
       {"BERT_Squad", "test data bug"},
       {"constantofshape_float_ones", "test data bug", {"onnx141", "onnx150"}},
       {"constantofshape_int_zeros", "test data bug", {"onnx141", "onnx150"}},
-      {"cast_STRING_to_FLOAT", "Linux CI has old ONNX python package with bad test data", {"onnx141"}},
+      {"cast_STRING_to_FLOAT", "Linux CI has old ONNX python package with bad test data", {}},
```

Comment:
ONNX test data may have bugs. For example, this one had a bug in onnx 1.4.1.  Then it gets fixed. This is why we only disabled it for onnx 1.4.1. Otherwise, we need to disable this test forever.